src/dbnode/storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -1425,6 +1421,72 @@ func (s commitLogSource) maybeAddToIndex(
 	return err
 }
 
+func (s *commitLogSource) availability(
```

Comment:
nit: mind adding 1-2 lines explaining intent behind the method -- i.e. that anything not initialized is by definition not available

Reply:
done